specifyweb/frontend/js_src/lib/components/Attachments/utils.ts

```diff
@@ -39,12 +39,18 @@ export const tablesWithAttachments = f.store(() =>
  */
 export const getAttachmentRelationship = (
   table: SpecifyModel
-): Relationship | undefined =>
-  table.name === 'Attachment'
-    ? undefined
-    : table.relationships.find((relationship) => {
+): Relationship | undefined => {
+  if (table.name === 'Attachment') return undefined;
+  const commonRelationship = table.field[`${table.name}Attachments`] as
```

Comment:
table.name is capitalized, but all fields are starting with lowercase, thus this will never match

so you would need something like `${table.name[0].toLowercase()}${table.name.slice(1)}Attachments`

Reply:
Fixed in https://github.com/specify/specify7/pull/4414/commits/2761272f9f3a2ead15f8e2a773714626409b908e